pkg/build/create.go

```diff
@@ -0,0 +1,10 @@
+package build
+
+type CreateBuild struct {
+	BuilderName      string            `help:"The name of the entity that triggered the build. Could be a user, system etc."`
+	Metadata         map[string]string `help:"(optional) Additional build information"`
```

Comment:
Are any of the flags _not_ optional? I'm not sure I see the distinction between those marked "(optional)" and those not. Perhaps we should just not show that in the help?